Examples/MAX32670/Temp_Monitor/temp_monitor.c

```diff
@@ -0,0 +1,267 @@
+#include "temp_monitor.h"
+
+#include <stdbool.h>
+#include <stdio.h>
+#include "flc.h"
+#include "i2c.h"
+#include "icc.h"
+#include "led.h"
+#include "lp.h"
+#include "max31889_driver.h"
+#include "mxc_device.h"
+#include "rtc.h"
+
+/***** Definitions *****/
+/***** Modifiable defines *****/
+#define HI_TEMP_THRESHOLD 30
+#define LO_TEMP_THRESHOLD 15
+#define TEMP_CHECK_PERIOD 5
+#define TR_STORAGE_PAGE \
+    1 //NOTE: Be sure this value is less than the total number of flash pages in the device.  \
```

Comment:
Weird clang-format.  Put 1 back up on the previous line and the comment below the `#define`